hydra-cardano-api/src/Hydra/Cardano/Api/ScriptData.hs

```diff
@@ -63,9 +63,10 @@ lookupScriptData (Tx (ShelleyTxBody _ _ _ scriptsData _ _) _) (TxOut _ _ datum _
     (TxOutDatumInline _ dat) ->
       Just dat
  where
-  datums = case scriptsData of
-    TxBodyNoScriptData -> mempty
-    TxBodyScriptData _ (Ledger.TxDats m) _ -> m
+  datums :: Map (Ledger.DataHash StandardCrypto) (Ledger.Data (ShelleyLedgerEra era)) =
```

Comment:
Is there a particular reason why you did use an inline type annotation over a type signature?